src/main/java/core/basesyntax/supliers/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.supliers;
+
+import core.basesyntax.interfaces.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        Random random = new Random();
```

Comment:
We can add field with type `Random` to our class, so we won't need to create an instance every time when method invoked